src/styles/main.scss

```diff
@@ -1,3 +1,231 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  line-height: 14px;
+}
+
+body,
+h2,
+p,
+ul {
   margin: 0;
 }
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header_nav {
+  font-weight: 500;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
you can reset these properties just like you reset margins and padding for body, h1, etc.